examples/tinkerbell/setup-proxy-on-nodes.md

```diff
@@ -0,0 +1,37 @@
+# Add proxy config to the `TinkerbellTempalteConfiguration`
+
+The `configure-containerd-proxy` action configures proxy for containerd on all the Kubernetes nodes. You can add the following section in your cluster config to 
+setup proxy on the nodes:
+
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: TinkerbellTemplateConfig
+metadata:
+  name: template-name
+spec:
+  template:
+    global_timeout: 6000
+    id: ""
+    name: template-name
+    tasks:
+    - actions:
+      // Append to existing actions.
+      - environment:
+          CONTENTS: |
+            [Service]
+            Environment:"HTTP_PROXY=<HTTP-PROXY-IP:PORT>"
+            Environment:"HTTPS_PROXY=<HTTPS-PROXY-IP:PORT>"
+            Environment:"NO_PROXY=<Comma-separated-list-of-no-proxies>"
+          DEST_DISK: /dev/sda2
```

Comment:
I would start putting comments near these DEST_DISK params saying this would change depending on the disk they have and also mention about partition 2